src/LanguageServer/Protocol/Features/Diagnostics/DiagnosticAnalyzerService.cs

```diff
@@ -70,7 +70,7 @@ public static bool IsGlobalOptionAffectingDiagnostics(IOption2 option)
                option == s_crashOnAnalyzerException;
 
         public void RequestDiagnosticRefresh()
-            => _diagnosticsRefresher?.RequestWorkspaceRefresh();
+            => _diagnosticsRefresher.RequestWorkspaceRefresh();
```

Comment:
as you can see, the call to DiagAnalyzerService.RequestDiagnosticRefresh is exactly equivalent to the prior calls to DiagRefreshService.RequestWorkspaceRefresh.

Note, this ?. was unnecessary as this field is mef imported in the constructor and thus cannot ever be null.